src/pages/talks/web-perf-101/index.tsx

```diff
@@ -2568,14 +2559,26 @@ const WebPerf101Page = ({ data }: WebPerf101PageProps) => {
             </p>
           </Slide>
         </Slides>
+        <TalkMeta
+          authors={[
+            {
+              description: 'PerfPerfPerf founder',
+              imageData: data.iamakulovPhoto.childImageSharp.fixed,
+              link: 'https://twitter.com/iamakulov',
+              name: 'Ivan Akulov',
+            },
+          ]}
+          lastUpdatedDate={lastUpdatedDate}
+        />
         <Footnote>
           <p>
             <strong>This talk was brought to you by PerfPerfPerf.</strong> We
             help companies to earn more by making web apps faster.
           </p>
```

Comment:
```suggestion
          <p>
            <strong>This talk was brought to you by PerfPerfPerf.</strong> We
            help companies to earn more by making web apps faster.
          </p>
```

Let’s remove this (sorry, I wasn’t clear that this has to go away as well), and it’s LGTM.